dizoo/bsuite/entry/memory_len/memory_len_0_dqn_main.py

```diff
@@ -0,0 +1,85 @@
+import os
+from tensorboardX import SummaryWriter
+
+from ding.config import compile_config
+from ding.worker import BaseLearner, SampleSerialCollector, InteractionSerialEvaluator, AdvancedReplayBuffer
+from ding.envs import BaseEnvManager
+from ding.policy import DQNPolicy
+from ding.model import DQN
+from ding.utils import set_pkg_seed
+from ding.rl_utils import get_epsilon_greedy_fn
+from dizoo.bsuite.envs import BSuiteEnv
+from dizoo.bsuite.config.serial.memory_len.memory_len_0_dqn_config import memory_len_0_dqn_config
+
+
+def main(cfg, seed=0):
+
+    def memory_len_env():
+        return BSuiteEnv(cfg)
+
+    cfg = compile_config(
+        cfg,
+        BaseEnvManager,
+        DQNPolicy,
+        BaseLearner,
+        SampleSerialCollector,
+        InteractionSerialEvaluator,
+        AdvancedReplayBuffer,
+        save_cfg=True
+    )
+    collector_env_num, evaluator_env_num = cfg.env.collector_env_num, cfg.env.evaluator_env_num
+    collector_env = BaseEnvManager(env_fn=[memory_len_env for _ in range(collector_env_num)], cfg=cfg.env.manager)
+    evaluator_env = BaseEnvManager(env_fn=[memory_len_env for _ in range(evaluator_env_num)], cfg=cfg.env.manager)
+
+    # Set random seed for all package and instance
+    collector_env.seed(seed)
+    evaluator_env.seed(seed, dynamic_seed=False)
+    set_pkg_seed(seed, use_cuda=cfg.policy.cuda)
+
+    # Set up RL Policy
+    model = DQN(**cfg.policy.model)
```

Comment:
Are we able to solve `memory_len/22` if not using any RNN model here?

Reply:
No, I tried but DQN could only solve memory_len/0 and memory_len/22. I haven't tried other policies yet.

Reply:
Okay.